test/load-generator/boulder-calls.go

```diff
@@ -335,9 +324,10 @@ func completeAuthorization(authz *core.Authorization, s *State, ctx *context) er
 	authStr := fmt.Sprintf("%s.%s", chalToSolve.Token, base64.RawURLEncoding.EncodeToString(thumbprint))
 
 	// Add the challenge response to the state's test server
+	fmt.Printf("\n\nADDING HTTP-01 for token %q keyauth %q\n\n", chalToSolve.Token, authStr)
 	s.challSrv.AddHTTPOneChallenge(chalToSolve.Token, authStr)
 	// Clean up after we're done
-	defer s.challSrv.DeleteHTTPOneChallenge(chalToSolve.Token)
+	//defer s.challSrv.DeleteHTTPOneChallenge(chalToSolve.Token)
```

Comment:
Is this leftover debug?

Reply:
I think I had to make this change to get `config-next` with multi VA working. I'll verify. Its hackish but I don't want to also rework the issuance flow in this PR to properly remove the challenge only when the authorization is complete not after the POST to initiate a challenge.

Reply:
Update: restoring the `defer`'d cleanup didn't break anything. Done in b2451c8.